packages/integrations/image/README.md

```diff
@@ -295,10 +297,10 @@ Source for the original image file.
 
 For images located in your project's `src`: use the file path relative to the `src` directory. (e.g. `src="../assets/source-pic.png"`)
 
-For images located in your `public` directory: use the URL path relative to the `public` directory. (e.g. `src="/images/public-image.jpg"`)
-
 For remote images, provide the full URL. (e.g. `src="https://astro.build/assets/blog/astro-1-release-update.avif"`)
 
+For images located in your `public` directory: use the URL path relative to the `public` directory. (e.g. `src="/images/public-image.jpg"`). These work like remote images.
```

Comment:
Same suggestion as above to keep `src/` & `public/` close by moving remote images first.

Reply:
done!